### PR TITLE
dont toggle search on shift + m if inside input or textarea

### PR DESCRIPTION
**Pull request by alexmorten**
https://trello.com/c/JN6gvkdj/3176-pivot-cant-enter-custom-duration

### CODE REVIEW

File: src/client/components/measures-tile/measures-tile.tsx [hunk 1]
```diff
@@ -103,8 +103,11 @@ export class MeasuresTile extends Component<MeasuresTileProps, MeasuresTileState
 
   private handleGlobalKeyDown = (e: KeyboardEvent) => {
     if (e.shiftKey && e.keyCode === keyCodes.m) {
-      e.preventDefault();
-      this.toggleSearch();
+      const element = e.target as HTMLElement;
```
Comment (cedrics) [inline src/client/components/measures-tile/measures-tile.tsx]: We have similar code for the `dimension` please add it there too (maybe extract the code for it, too) => https://github.com/remerge/turnilo/blob/master/src/client/components/dimension-list-tile/dimension-list-tile.tsx#L178-L183. Something like:

```js
if (originatesFromTextAreaOrInput(e)) return;
```

File: src/client/utils/dom/dom.ts [hunk 2]
```diff
@@ -165,3 +165,11 @@ export function classNames(...args: Array<string | Record<string, any>>): string
 
   return classes.join(" ");
 }
+
+export function originatesFromTextAreaOrInput(e: KeyboardEvent): boolean {
+  const element = e.target as HTMLElement;
+  if (element.tagName === "INPUT" || element.tagName === "TEXTAREA") {
```
Comment (cedrics) [inline src/client/utils/dom/dom.ts]: This could just be `return element.tagName === "INPUT" || element.tagName === "TEXTAREA"`

Comment (alexmorten) [reply src/client/utils/dom/dom.ts]: That should have occurred to me 😅

File: src/client/components/measures-tile/measures-tile.tsx [hunk 3]
```diff
@@ -173,7 +175,7 @@ export class MeasuresTile extends Component<MeasuresTileProps, MeasuresTileState
     return dataCube.measures.accept(measuresConverter).filter(item => !searchText || item.hasSearchText || item.type === MeasureForViewType.group);
   }
 
-    render() {
+render() {
```
Comment (wmadden) [inline src/client/components/measures-tile/measures-tile.tsx]: You've incorrectly corrected the indentation here

**Pull request merged**